gcp/compute/main.tf

```diff
@@ -0,0 +1,35 @@
+data "google_compute_zones" "available" {}
+
+variable project_id {}
+
+resource "google_project_service" "project" {
+  project = "${var.project_id}"
+  service = "compute.googleapis.com"
+
+  disable_dependent_services = true
```

Comment:
Why do we need this?

Reply:
while true, it disables all depedencies that were enabled for this api to work.